include/boost/geometry/strategies/spherical/area_huiller.hpp

```diff
@@ -142,20 +150,14 @@ public :
 
             E = geometry::math::abs(E);
 
-            // In right direction: positive, add area. In left direction: negative, subtract area.
-            // Longitude comparisons are not so obvious. If one is negative, other is positive,
-            // we have to take the dateline into account.
```

Comment:
Sorry, I got an impression that it's a comment made explicitly for the approach used below. But indeed it's more general.